src/Metalama.Framework.RunTime.Tests/Metalama.Framework.RunTime.Tests.csproj

```diff
@@ -6,7 +6,7 @@
 
     <IsPackable>false</IsPackable>
     <IsTestProject>true</IsTestProject>
-    <NoWarn>SA0001</NoWarn>
+    <NoWarn>SA0001;NU1903</NoWarn>
```

Comment:
Comment on what these warnings mean, why they are suppressed, and when (if ever) this suppression should be removed.